README.md

```diff
@@ -1,89 +1,216 @@
+Setting up Log backend ... <br/>
 
-1) Build the API Server container image:
+If not in kubernetes directory... <br/>
+cd $GOPATH/src/k8s.io/kubernetes <br/>
 
-   `$ ./build-provenance-artifacts.sh`
+vi hack/local-up-cluster.sh <br/>
+
+line 87: Change ENABLE_APISERVER_BASIC_AUDIT to true
+   ENABLE_APISERVER_BASIC_AUDIT=${ENABLE_APISERVER_BASIC_AUDIT:-true}
+
+line 486: add audit-policy file to audit_args:   
+   Now you need to add an audit-arg for the audit-policy. add the following line after audit_arg+=" --audit-log-maxbackup=0"
+
+   audit_arg += " --audit-policy-file=/root/audit-policy.yaml" <br/>
+
+   The value of --audit-policy-file is where you created your audit-policy.yaml file.  <br/>
+   There is an example-policy for a postgres custom resource saved in this repository. <br/>
+
+   Note: the audit log for your custom resource will be saved where this variable is set:
+      APISERVER_BASIC_AUDIT_LOG=/tmp/kube-apiserver-audit.log
+
+
+   This file defines what actions and resources will generate logs.
+
+   An example of a audit-policy file: reference the docs if you are looking to make one: <br/>
+      https://kubernetes.io/docs/tasks/debug-application-cluster/audit/
+
+   For running kubeprovenance to track only a postgres custom resource, audit-policy would look like this:  <br/>
+   Add more rules to the audit-policy to track different or more than one custom resource:
+
+      root@provenance:~# more audit-policy.yaml
+      apiVersion: audit.k8s.io/v1beta1
+      kind: Policy
+      omitStages:
+        - "RequestReceived"
+      rules:
+        - level: Request
+          verbs:
+            - create
+            - delete
+            - patch
+          resources:
+            - group: "postgrescontroller.kubeplus"
+              version: "v1"
+              resources: ["postgreses"]
+
+   Note: our approach may change to a webhook backend instead of a log backend <br/>
+
+
+**8. Running kubeprovenance** <br/>
+
+Install dep:  <br/>
+curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh <br/>
+
+git clone https://github.com/cloud-ark/kubeprovenance.git <br/>
+mv kubeprovenance $GOPATH/src/github.com/cloud-ark <br/>
+cd $GOPATH/src/github.com/cloud-ark/kubeprovenance <br/>
+dep ensure <br/>
 
-2) Deploy the API Server in your cluster:
+Make sure kubernetes is running:
+$ kubectl.sh cluster-info
 
+Now to deploy this aggregated api server use these commands:
+1) Build the API Server container image:  <br/>
+   `$ ./build-provenance-artifacts.sh`
+2) Deploy the API Server in your cluster: <br/>
    `$ ./deploy-provenance-artifacts.sh`
+3) Clean-up:  <br/>
+   `$ ./delete-provenance-artifacts.sh`
+
+Test using these following commands:   
+1) Get list of versions for client25 postgres
+
+```
+kubectl.sh get --raw "/apis/kubeprovenance.cloudark.io/v1/namespaces/default/postgreses/client25/versions"
+```
+
+2) Get Spec history for client25 postgres
```

Comment:
Get Spec history for Postgres custom resource

Reply:
Ok I committed these changes